### PR TITLE
[armpl-gcc] Make pkg-config files available

### PR DESCRIPTION
**Pull request by stephenmsachs**
ARMpl pkgconfig files are located in a non-default location and do not have the .pc extension. Changing those both helps pkgconfig pick them up correctly, e.g. in the meson build of `py-scipy`.

### CODE REVIEW

File: var/spack/repos/builtin/packages/armpl-gcc/package.py [hunk 1]
```diff
@@ -262,3 +261,15 @@ def check_install(self):
         make("-C", armpl_example_dir, "ARMPL_DIR=" + armpl_dir)
         # clean up
         make("-C", armpl_example_dir, "ARMPL_DIR=" + armpl_dir, "clean")
+
+    @run_after("install")
+    def make_pkgconfig_files(self):
+        # ARMpl pkcfonfig files do not have .pc extension and are thus not found by pkg-config
```
Comment (annop-w) [inline var/spack/repos/builtin/packages/armpl-gcc/package.py]: nit: # ArmPL pkgconfig

File: var/spack/repos/builtin/packages/armpl-gcc/package.py [hunk 2]
```diff
@@ -262,3 +261,15 @@ def check_install(self):
...
+        armpl_dir = get_armpl_prefix(self.spec)
+        for f in find(join_path(armpl_dir, "pkgconfig"), "*"):
+            copy(f, f + ".pc")
```
Comment (annop-w) [inline var/spack/repos/builtin/packages/armpl-gcc/package.py]: Is it better to create symlink instead ?

**Review by annop-w — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/armpl-gcc/package.py (hunk 1 above)
Comment (stephenmsachs) [reply var/spack/repos/builtin/packages/armpl-gcc/package.py]: thanks.

File: var/spack/repos/builtin/packages/armpl-gcc/package.py (hunk 2 above)
Comment (stephenmsachs) [reply var/spack/repos/builtin/packages/armpl-gcc/package.py]: symlinks are probably clearer.

**Review by annop-w — APPROVED** [review]

**Review by annop-w — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**